packages/mdc-select/README.md

```diff
@@ -403,12 +425,16 @@ First, wrap both a custom select and a native select within a wrapper element, l
 <div class="select-manager">
```

Comment:
Can we test that this example still actually works?  (See #1025)

Reply:
We may need to update the MDC Select example shown here, but what makes you think this wouldn't work anymore?